vignettes/bcdata.Rmd

```diff
@@ -0,0 +1,186 @@
+---
+
+
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  warning = FALSE,
+  message = FALSE
+)
+
+library(bcdata)
+library(ggplot2)
+```
+
+The `bcdata` [R](https://www.r-project.org/) package contains functions for searching & retrieving data from the [B.C. Data Catalogue]( https://catalogue.data.gov.bc.ca).
+
+The [B.C. Data Catalogue](https://www2.gov.bc.ca/gov/content?id=79B5224167334667A44C9E8B5143D0C5) is the place to find British Columbia Government data, applications and web services. Much of the data are released under the [Open Government Licence --- British Columbia](https://www2.gov.bc.ca/gov/content/data/open-data/open-government-licence-bc), as well as numerous other [licences](https://catalogue.data.gov.bc.ca/dataset?download_audience=Public).
+
+  
+  
+You can install `bcdata` directly from GitHub using the [remotes](https://cran.r-project.org/package=remotes) package:
+
+```{r, eval=FALSE}
+install.packages("remotes")
+
+remotes::install_github("bcgov/bcdata")
+library(bcdata)
+```
+
+### `bcdc_browse()`
+
+`bcdata::bcdc_browse()` let's you access the B.C. Data Catalogue web interface directly from R---opening the home page in your default browser:
+
+```{r, eval=FALSE}
+## Take me to the B.C. Data Catalogue home page
+bcdc_browse()
+```
+
+If you know the catalogue record "human-readable" name or permanent ID you can open directly to the record web page:
+
+```{r, eval=FALSE}
+## Take me to the B.C. Winery Locations catalogue record using the record name
+bcdc_browse("bc-winery-locations") 
+
+## Take me to the B.C. Winery Locations catalogue record using the record ID
+bcdc_browse("1d21922b-ec4f-42e5-8f6b-bf320a286157")
+```
+
+You can also use `bcdc_browse()` to open to search results in the catalogue:
+
+```{r, eval=FALSE}
+## Take me to the catalogue search results for 'wineries'
+bcdc_browse("wineries")
+```
+
+### `bcdc_search()`
+
+`bcdata::bcdc_search()` let's you search records in the B.C. Data Catalogue, returning the search results in your R session. 
+
+Let's search the catalogue for records that contain the word "recycling":
+
+```{r}
+## Give me the catalogue search results for 'recycling'
+bcdc_search("recycling")
+```
+
+You can set the number of records to be returned from the search and/or you can customize your search using the catalogue search facets `license_id`, `download_audience`, `type`, `res_format`, `sector`, and `organization`:
+
+```{r}
+## Give me the first catalogue search result for 'recycling'
+bcdc_search("recycling", n = 1)
+
+## Give me the catalogue search results for 'recycling' where the 
+## data is tabular and licenced under Open Government Licence – British Columbia
+bcdc_search("recycling", type = "Dataset", license_id = "2")
+```
+
+You can see all valid values for the catalogue search facets using `bcdata::bcdc_search_facets()`:
+
+```{r}
+## Valid values for search facet 'type'
+bcdc_search_facets(facet = "type")
+```
```

Comment:
I wonder if an additional example with licenses would be helpful here, especially since licenses are specified by number?

Reply:
Or maybe just change to the licence example? I'll try it out.